Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -291,10 +292,19 @@ - (void)launchApplicationWithContext:(BPExecutionContext *)context {
 
 - (void)checkProcessWithContext:(BPExecutionContext *)context {
     BOOL isRunning = [self isProcessRunningWithContext:context];
-    if (!isRunning && [context.runner isFinished]) {
-        [[BPStats sharedStats] endTimer:RUN_TESTS(context.attemptNumber)];
-        [self runnerCompletedWithContext:context];
-        return;
+    if (!isRunning) {
+        if ([context.runner isFinished]) {
+            [[BPStats sharedStats] endTimer:RUN_TESTS(context.attemptNumber)];
+            [self runnerCompletedWithContext:context];
+            return;
+        } else {
+            // App crashed
+            [[BPStats sharedStats] addSimulatorLaunchFailure];
```

Comment:
This is not simulator launch failure but app launch failure, create another entry for this case?